src/main/java/appeng/block/storage/BlockSkyChest.java

```diff
@@ -39,24 +34,19 @@
 import net.minecraft.util.math.AxisAlignedBB;
 import net.minecraft.util.math.BlockPos;
 import net.minecraft.world.World;
-import net.minecraftforge.fml.relauncher.Side;
-import net.minecraftforge.fml.relauncher.SideOnly;
 
 import appeng.api.util.AEPartLocation;
 import appeng.block.AEBaseTileBlock;
-import appeng.block.IHasSpecialItemModel;
-import appeng.client.render.tesr.SkyChestTESR;
-import appeng.core.features.AEFeature;
 import appeng.core.sync.GuiBridge;
 import appeng.helpers.ICustomCollision;
 import appeng.tile.storage.TileSkyChest;
 import appeng.util.Platform;
 
 
-public class BlockSkyChest extends AEBaseTileBlock implements ICustomCollision, IHasSpecialItemModel
+public class BlockSkyChest extends AEBaseTileBlock implements ICustomCollision
 {
 
-	public static enum SkyChestType
+	public enum SkyChestType
```

Comment:
To be honest, I just saw it because IntelliJ was complaining about a redundant specfier hehe